service/orchestrator/audit_scope.go

```diff
@@ -29,20 +29,32 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"slices"
+	"strings"
 
 	"clouditor.io/clouditor/v2/api"
 	"clouditor.io/clouditor/v2/api/orchestrator"
 	"clouditor.io/clouditor/v2/internal/logging"
 	"clouditor.io/clouditor/v2/persistence"
-	"clouditor.io/clouditor/v2/persistence/gorm"
 	"clouditor.io/clouditor/v2/service"
+	"github.com/google/uuid"
 	"github.com/sirupsen/logrus"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 	"google.golang.org/protobuf/types/known/emptypb"
 )
 
+// ErrAuditScopeNotFound indicates the audit scope was not found
+var ErrAuditScopeNotFound = status.Error(codes.NotFound, "audit scope not found")
```

Comment:
wasn't this already defined somewhere else?

Reply:
Yes, it is also defined in service/orchestrator/audit_scope.go:l48. I didn't want to include the service packages in another one. Perhaps we could move the error messages to the API folder instead?!? I'm not really happy with the error definitions anyway, we should harmonize them.

Reply:
isn't there a api/errors.go?

Reply:
I tried to consolidate it, but now some tests fail